setup.py

```diff
@@ -200,6 +214,8 @@ def get_extension_modules():
             pyarrow_module_link_args.append("-Wl,-rpath,@loader_path/pyarrow")
         else:
             pyarrow_module_link_args.append("-Wl,-rpath,$ORIGIN/pyarrow")
+            if not os.environ.get("CONDA_BUILD") and is_cxx11_abi():
```

Comment:
I'm a bit confused about the condition here. We add `ABI=0` for the case that `is_cxx11_abi` is true. Shouldn't it be the other way around?

Weirdly the tests pass :D

Reply:
That's because for whatever reasons the arrow libraries built for pyarrow are built with ABI=0. So turbodbc would build but the arrow support is broken as described in #375

Reply:
Yes, `pyarrow` `manylinux` wheels are built with `ABI=0`. But you set `_GLIBCXX_USE_CXX11_ABI=0` in the case that `_GLIBCXX_USE_CXX11_ABI` is set to `1` in the compiler output. This is a bit contradicting.

Reply:
Yes, it is confusing and it is all because of https://gcc.gnu.org/onlinedocs/libstdc++/manual/using_dual_abi.html. Basically, if we compile turbodbc with _GLIBCXX_USE_CXX11_ABI set to 1, then any declaration of std::string becomes std::__cxx11:string and at runtime the linker won't be able to resolve any arrow methods with std::string in the definitions.